src/apps/distributed_efforts/services/rating_network_pairer.py

```diff
@@ -130,9 +131,25 @@ def _choose_opponent(self, reference_network):
                 return None
             return np.random.choice(nearby_networks)
 
-        win_probability = np.asarray([
-            1 / (1 + exp((opp_net.log_gamma - ref_net_log_gamma) / self.current_run.rating_game_entropy_scale)) for opp_net in nearby_networks
-        ])
-        shannon_entropy = -win_probability * np.log2(win_probability) - (1 - win_probability) * np.log2(1 - win_probability)
+        log_variances = [
+            self._log_variance_of_gamma_difference((opp_net.log_gamma - ref_net_log_gamma) / self.current_run.rating_game_variability_scale) for opp_net in nearby_networks
+        ]
+        max_log_variance = max(log_variances)
+        # Subtract out the max to make sure that we're near 0, for numerical stability
+        log_variances = [ log_variance - max_log_variance for log_variance in log_variances ]
+        variances = [ exp(log_variance) for log_variance in log_variances ]
+        print(log_variances)
```

Comment:
Maybe remove the print.

ideally use logging in debug (we need more logs on the server)

Reply:
Oops hehe good catch on the print.

Reply:
Wait, I did at least comment out the print lat in a later commit I think. Seems okay now.